final_task/rss_reader/logs.py

```diff
@@ -0,0 +1,191 @@
+"""This module creates logs, writes logs, reads logs."""
+
+import logging
+import os
+import news
+
+this_directory = os.path.abspath(os.path.dirname(__file__))
+
+journalLog = 'logJournal.log'
+directory = os.path.join(this_directory, journalLog)
```

Comment:
Same about global constants

Reply:
All global variables were fixed, it will be updated during the next commit.